Week-6/6.4/assignment6.4.js

```diff
@@ -0,0 +1,68 @@
+function returnMaxProfit(arr, quantity) {
+    let mainArr = arr;
+    let arrLen = mainArr.length;
+    // console.log(arrLen);
+
+    let maxProfit = 0;
+
+    while(arrLen > 2) {
```

Comment:
@mr-rego why is this while loop needed ? any reasons ?

Why is the quantity needed, its not a input given to us. Can you change this a bit without quantity ?